docker/config/test.env

```diff
@@ -21,7 +21,8 @@ resource.boto.bucket_name=crashstats-test
 TELEMETRY_BUCKET_NAME=telemetry-test
 
 # Elasticsearch configuration
-ELASTICSEARCH_URLS=http://example.com:9123
+resource.elasticsearch.elasticsearch_index=testsocorro%Y%W
+resource.elasticsearch.elasticsearch_index_regex=^testsocorro[0-9]{6}$
```

Comment:
This prevents the tests from stomping on data in the local dev environment which is kind of annoying.